ras/ras_tools.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM
+# Author: Shirisha Ganta <shirisha.ganta1@ibm.com>
+
+from avocado import Test
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+class Ras_tools(Test):
+    """
+    :avocado: tags=privileged
+    """
+    fail_cmd = list()
+
+    def run_cmd(self, cmd):
+        cmd_result = process.run(cmd, ignore_status=True, sudo=True,
+                                 shell=True)
+        if cmd_result.exit_status != 0:
```

Comment:
may just use this way 

```
if process.system(cmd, ignore_status=True, sudo=True, shell=True):
   self.fail_cmd.append(cmd)
return
```

this way we can avoid using extra variable

Reply:
sure will do that